### PR TITLE
accessibility: issue/#2083 added completion status to headers

### PR DESCRIPTION
**Pull request by oliverfoster**
#2083 
* Removed accessibility state from after component
* Added headings manager and headingView to draw accessibility headers

### CODE REVIEW

**Comment by chris-steele** [comment]
@oliverfoster good work! 👍

File: src/core/templates/heading.hbs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{{import_globals}}
+<div id="{{_id}}-heading" class="js-heading-inner" {{{a11y_attrs_heading _type}}} >
+    <span class="aria-label">
+    {{#if _isComplete}}
+        {{_globals._accessibility._ariaLabels.complete}} {{compile displayTitle this}}
```
Comment (moloko) [inline src/core/templates/heading.hbs]: it's not necessary to have `this` in `{{compile displayTitle this}}`

File: src/core/js/headings.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        },
+
+        onViewRender: function(view) {
+            var $headingSeats = view.$('.js-heading');
+            $headingSeats.each(function(index, el) {
+                new HeadingView({
+                    el: el,
+                    model: view.model
+                });
+            }.bind(this));
```
Comment (moloko) [inline src/core/js/headings.js]: I don't think you need `.bind(this)` here?

**Comment by moloko** [comment]
@oliverfoster you has conflict

**Pull request merged**